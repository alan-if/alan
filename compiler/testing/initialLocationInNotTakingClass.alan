The l Isa location
End The l.

Every subobject1 Isa object
End Every.

Every subobject2 Isa object
End Every.

The cont Isa object At l
  Container Taking subobject1.
End The cont.

The t Isa subobject2 In cont
End The t.

Start At l.
