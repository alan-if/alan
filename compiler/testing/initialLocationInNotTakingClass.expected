########## initialLocationInNotTakingClass ##########


initialLocationInNotTakingClass.alan

   14.  The t Isa subobject2 In cont
=====>                          1

  *1*   404 E : Container is restricted to only contain instances of
                'subobject1' and its subclasses.


        1 error(s).
        No detected warnings.
        1 informational message(s).

